hendrix/test/resources.py

```diff
@@ -17,82 +17,74 @@ class TestNameSpace(object):
     async_task_was_run = False
     ready_to_proceed_with_second_cycle = Queue()
     second_cycle_complete = Queue()
-    
+
 nameSpace = TestNameSpace()
 
 
 def application(environ, start_response):
-    start_response('200 OK', [('Content-type','text/plain')])
-    
+    start_response('200 OK', [('Content-type', 'text/plain')])
+
     if 'test_crosstown_traffic' in environ['QUERY_STRING'] or environ['PATH_INFO'] == '/r/1':
-        logger.info("Starting first cycle.")
```

Comment:
I'm not a big fan of having log statements being emitted to the terminal during tests. But I understand what you mean. I'll put in a `log.debug` statement and later we can add a log analysis utility so to enable to easily resolve/find related test issues.